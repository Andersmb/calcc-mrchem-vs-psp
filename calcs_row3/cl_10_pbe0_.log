Starting job 5688107 on c1-[27-28,30,53],c5-2 at Thu May 19 19:54:01 CEST 2022

Copying cl_10_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/singlets
Copying cl_10_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/singlets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688107      cl_10_pbe+        180                                             00:57:09      0:0 
5688107.bat+      batch         36        1   00:00:02          0   00:00:02   00:57:09      0:0 
5688107.ext+     extern        180        5   00:00:00          4   00:00:00   00:57:09      0:0 
5688107.0      mrchem.x        180       30   00:56:26         29   02:42:59   00:56:42      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688107                                                                          
5688107.bat+     19280K          0     19280K        0              0          0 
5688107.ext+          0          4          0        0              4          0 
5688107.0     13817436K         29 5381992994        0             21          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688107                                                                                               
5688107.bat+       19.39M               0         19.39M        0.07M                0          0.07M 
5688107.ext+        0.00M               4          0.00M            0                4              0 
5688107.0           6.77M              17          6.74M      125.87M                0         67.53M 

Job 5688107 completed at Thu May 19 20:51:10 CEST 2022
