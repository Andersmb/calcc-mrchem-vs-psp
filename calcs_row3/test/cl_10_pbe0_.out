                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       F            :        0.000000       -1.476178       -2.343765
    1      Cl            :        0.000000        0.880310        0.033699
    2       O            :        0.000000       -0.197779        2.708418
---------------------------------------------------------------------------
 Center of mass          :        0.000000       -0.000000        0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              17
 Beta electrons          :                                              17
 Total electrons         :                                              34
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999998625253e-01
    1    1    a          :                              9.999999900145e-01
    2    1    a          :                              9.999998681194e-01
    3    1    a          :                              1.000000009181e+00
    4    1    a          :                              9.999999990521e-01
    5    1    a          :                              9.999999952022e-01
    6    1    a          :                              1.000000001025e+00
    7    1    a          :                              9.999997575253e-01
    8    1    a          :                              1.000000168717e+00
    9    1    a          :                              1.000000159820e+00
   10    1    a          :                              1.000000139343e+00
   11    1    a          :                              9.999999036187e-01
   12    1    a          :                              9.999999926866e-01
   13    1    a          :                              1.000000020382e+00
   14    1    a          :                              9.999999788227e-01
   15    1    a          :                              9.999999706714e-01
   16    1    a          :                              1.000000036890e+00
   17    1    b          :                              9.999999947476e-01
   18    1    b          :                              9.999999584278e-01
   19    1    b          :                              9.999998681191e-01
   20    1    b          :                              1.000000013244e+00
   21    1    b          :                              9.999999994905e-01
   22    1    b          :                              9.999999952020e-01
   23    1    b          :                              1.000000000108e+00
   24    1    b          :                              9.999997499854e-01
   25    1    b          :                              1.000000169584e+00
   26    1    b          :                              1.000001742661e+00
   27    1    b          :                              1.000000139364e+00
   28    1    b          :                              9.999999023106e-01
   29    1    b          :                              9.999999930387e-01
   30    1    b          :                              1.000000025952e+00
   31    1    b          :                              9.999999788691e-01
   32    1    b          :                              1.000000065345e+00
   33    1    b          :                              1.000000029796e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                629.031965988136
 E-N energy              :            (au)              -1713.006381014639
 Coulomb energy          :            (au)                392.876159644974
 Exchange energy         :            (au)                -11.439935909400
 X-C energy              :            (au)                -35.479412834169
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                106.682096928355
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -738.017604125097
 Nuclear energy          :            (au)                106.682096928355
---------------------------------------------------------------------------
 Total energy            :            (au)             -6.313355071967e+02
                         :      (kcal/mol)             -3.961690088819e+05
                         :        (kJ/mol)             -1.657571134238e+06
                         :            (eV)             -1.717951358207e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)               -101.501344982065
    1    1    a          :            (au)                -24.681943072575
    2    1    a          :            (au)                -18.656578024119
    3    1    a          :            (au)                 -9.896398550694
    4    1    a          :            (au)                 -7.647908460360
    5    1    a          :            (au)                 -7.641404128243
    6    1    a          :            (au)                 -7.628267734082
    7    1    a          :            (au)                 -1.282880045640
    8    1    a          :            (au)                 -1.113033261547
    9    1    a          :            (au)                 -0.801205985765
   10    1    a          :            (au)                 -0.605112676255
   11    1    a          :            (au)                 -0.591874773862
   12    1    a          :            (au)                 -0.463908487856
   13    1    a          :            (au)                 -0.436931594324
   14    1    a          :            (au)                 -0.296328978824
   15    1    a          :            (au)                 -0.146272923018
   16    1    a          :            (au)                 -0.062603615092
   17    1    b          :            (au)               -101.501484578496
   18    1    b          :            (au)                -24.681983889639
   19    1    b          :            (au)                -18.656578074417
   20    1    b          :            (au)                 -9.896400559111
   21    1    b          :            (au)                 -7.647908285269
   22    1    b          :            (au)                 -7.641403722157
   23    1    b          :            (au)                 -7.628267587374
   24    1    b          :            (au)                 -1.282885140791
   25    1    b          :            (au)                 -1.113032748534
   26    1    b          :            (au)                 -0.801202315808
   27    1    b          :            (au)                 -0.605115905831
   28    1    b          :            (au)                 -0.591852051722
   29    1    b          :            (au)                 -0.463907387105
   30    1    b          :            (au)                 -0.436920481834
   31    1    b          :            (au)                 -0.296329090837
   32    1    b          :            (au)                 -0.146273189418
   33    1    b          :            (au)                 -0.062562266107
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -366.908104568772
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          5.830952e+00        -631.335507196743         -6.313355e+02
    1          4.923639e-01        -634.661759433915         -3.326252e+00
    2          2.524479e-01        -634.818019215081         -1.562598e-01
    3          1.466917e-01        -634.849279557529         -3.126034e-02
    4          7.167767e-02        -634.862247604887         -1.296805e-02
    5          7.717990e-02        -634.873632452143         -1.138485e-02
    6          2.947125e-02        -634.875828927456         -2.196475e-03
    7          3.294313e-02        -634.877715515182         -1.886588e-03
    8          2.326888e-02        -634.878269721192         -5.542060e-04
    9          2.878432e-02        -634.878752857416         -4.831362e-04
   10          1.597661e-02        -634.879064344248         -3.114868e-04
   11          1.198451e-02        -634.879247946817         -1.836026e-04
   12          6.776830e-03        -634.879297610523         -4.966371e-05
   13          2.960298e-03        -634.879314860725         -1.725020e-05
   14          2.214653e-03        -634.879324906294         -1.004557e-05
   15          2.304902e-03        -634.879332236654         -7.330361e-06
   16          1.632127e-03        -634.879334902078         -2.665423e-06
   17          9.220651e-04        -634.879335998554         -1.096476e-06
   18          6.499258e-04        -634.879336672000         -6.734459e-07
   19          5.785877e-04        -634.879337104019         -4.320189e-07
   20          6.272381e-04        -634.879337431248         -3.272289e-07
   21          7.198500e-04        -634.879337724844         -2.935958e-07
   22          7.117781e-04        -634.879337999772         -2.749283e-07
   23          4.299220e-04        -634.879338133345         -1.335727e-07
   24          4.465657e-04        -634.879338260731         -1.273867e-07
   25          4.231144e-04        -634.879338379832         -1.191005e-07
   26          3.072174e-04        -634.879338450643         -7.081110e-08
   27          2.723829e-04        -634.879338503870         -5.322693e-08
   28          2.563059e-04        -634.879338547499         -4.362869e-08
   29          1.921698e-04        -634.879338577531         -3.003254e-08
   30          1.497145e-04        -634.879338594071         -1.653950e-08
---------------------------------------------------------------------------
                      SCF converged in 30 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       F            :        0.000000       -1.476178       -2.343765
    1      Cl            :        0.000000        0.880310        0.033699
    2       O            :        0.000000       -0.197779        2.708418
---------------------------------------------------------------------------
 Center of mass          :        0.000000       -0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                633.566066918321
 E-N energy              :            (au)              -1722.597843787336
 Coulomb energy          :            (au)                394.296768241004
 Exchange energy         :            (au)                -11.397426261781
 X-C energy              :            (au)                -35.429000632633
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                106.682096928355
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -741.561435522426
 Nuclear energy          :            (au)                106.682096928355
---------------------------------------------------------------------------
 Total energy            :            (au)             -6.348793385941e+02
                         :      (kcal/mol)             -3.983927966402e+05
                         :        (kJ/mol)             -1.666875462225e+06
                         :            (eV)             -1.727594614277e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)               -101.902363898357
    1    1    a          :            (au)                -24.803221934241
    2    1    a          :            (au)                -19.292518607952
    3    1    a          :            (au)                 -9.711750050963
    4    1    a          :            (au)                 -7.448267525555
    5    1    a          :            (au)                 -7.443244674074
    6    1    a          :            (au)                 -7.433080176305
    7    1    a          :            (au)                 -1.239580651658
    8    1    a          :            (au)                 -1.144460184403
    9    1    a          :            (au)                 -0.836340812665
   10    1    a          :            (au)                 -0.575239066888
   11    1    a          :            (au)                 -0.535638783023
   12    1    a          :            (au)                 -0.523005601628
   13    1    a          :            (au)                 -0.447316794086
   14    1    a          :            (au)                 -0.443036704143
   15    1    a          :            (au)                 -0.396182612775
   16    1    a          :            (au)                 -0.308578109849
   17    1    b          :            (au)               -101.902363899554
   18    1    b          :            (au)                -24.803221976027
   19    1    b          :            (au)                -19.292518575338
   20    1    b          :            (au)                 -9.711750048372
   21    1    b          :            (au)                 -7.448267526684
   22    1    b          :            (au)                 -7.443244667402
   23    1    b          :            (au)                 -7.433080177289
   24    1    b          :            (au)                 -1.239580728571
   25    1    b          :            (au)                 -1.144460157830
   26    1    b          :            (au)                 -0.836340773921
   27    1    b          :            (au)                 -0.575239117427
   28    1    b          :            (au)                 -0.535638749530
   29    1    b          :            (au)                 -0.523005610803
   30    1    b          :            (au)                 -0.447316823470
   31    1    b          :            (au)                 -0.443036723628
   32    1    b          :            (au)                 -0.396182556541
   33    1    b          :            (au)                 -0.308578097124
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -368.967652398078
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.000004        0.638016       -1.324193
 Magnitude               :            (au)                        1.469881
                         :         (Debye)                        3.736065
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.097438        1.146346
 Magnitude               :            (au)                        1.150480
                         :         (Debye)                        2.924228
---------------------------------------------------------------------------
 Total vector            :        0.000004        0.735454       -0.177847
 Magnitude               :            (au)                        0.756652
                         :         (Debye)                        1.923216
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 56m 29s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
